docs(readme): Fixes formatting of publishing options table in the docs

errolpais opened the pull request:
Fixing some typos I found in the docs.

mefellows commented:
Thanks. Can you please describe/show what has actually changed in the publishing options? I'm finding the diff hard to see. It doesn't look like anything has changed to my eye, but I suspect it's some small detail.

errolpais commented:
I just discovered github's rich diff feature. Its great for viewing diffs in markdown.
In the contributing file, I added  **be** in "we don't want to _be that person_"
![image](https://user-images.githubusercontent.com/2432170/58726877-bca8f300-8397-11e9-9ae8-b43fd2bd970b.png)

In the Readme, the table for publishing options was broken since it had a newline in it, I fixed that and aligned the table in md form.
![image](https://user-images.githubusercontent.com/2432170/58726948-e5c98380-8397-11e9-9368-b7d28d4cb4b4.png)
![image](https://user-images.githubusercontent.com/2432170/58726971-f679f980-8397-11e9-9238-171bbda12d75.png)

mefellows commented:
Nice! thanks, merging.
